docs/management/api/blocklist-api.asciidoc

```diff
@@ -0,0 +1,614 @@
+:endpoint-artifact-list-id:                     endpoint_blocklists
+:endpoint-artifact-name:                        blocklist entries
+:endpoint-artifact-name-single:                 blocklist entry
+:endpoint-artifact-name-sentence-start:         Blocklist entries
+:endpoint-artifact-name-single-sentence-start:  Blocklist entry
```

Comment:
There was a discussion about it, and finally we decided to use the word `entry` to define if it's plural or not since a Blocklist means a list of items (plural). I think @caitlinbetz can give use a better context of this.

Reply:
The page title should be `Blocklist` to be consistent with the way it's displayed in the UI.  And yes, I remember settling on "entry" as a means to make it plural or not.

Reply:
FYI, I've changed the page title: https://github.com/elastic/security-docs/pull/1880/commits/c45fb0549c2203d323a166d27d78fc88bd54c67c